src/runtimes/node/bundlers/esbuild/bundler.ts

```diff
@@ -84,11 +87,21 @@ export const bundleJsFile = async function ({
   // URLs, not paths, so even on Windows they should use forward slashes.
   const sourceRoot = targetDirectory.replace(/\\/g, '/')
 
+  // Configuring the output format of esbuild. The `includedFiles` array we get
```

Comment:
We were not including a `package.json` when bundling with esbuild. This becomes a problem with ESM functions, since an ESM file and a lack of a `package.json` with `{"type": "module"}` would lead to a CJS function with ESM syntax.